### DISCUSSION TITLE
Solid JS tutorial is broken?

### DISCUSSION THREAD

**Discussion by Lirrrx** [solidjs/solid#1659] [General]
I have the error in console on the every https://www.solidjs.com/tutorial/ page: 'The requested module 'https://cdn.skypack.dev/solid-js' does not provide an export ...' 
For example.
page: https://www.solidjs.com/tutorial/lifecycles_onmount
error in repl console: The requested module 'https://cdn.skypack.dev/solid-js' does not provide an export named 'Match'
Is it the bug or only my problem?

**Comment by atk** [comment]
This is related to https://github.com/solidjs/solid/issues/1660.
